content/discrete-math/discrete-math.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: 'Discrete Math'
+Description: 'Discrete math is a fundamental branch of mathematics that provides the foundation for various concepts and techniques used in computer science and problem-solving.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/computer-science'
```

Comment:
This should have subjects and tags instead of this link

Reply:
The subjects are mentioned in catalog content right. I referred to other docs and they have similar structure.

Reply:
I have added subjects and tags to it.

Reply:
@caupolicandiaz, that's wrong because this is a topic page. It needs to follow [this template](https://github.com/Codecademy/docs/blob/main/documentation/topic-entry-template.md).

Reply:
My mistake